dbt.cmd

```diff
@@ -1,19 +1,17 @@
 @echo off
-call "%~dp0scripts\toolchain\dbtenv.cmd" env
+
```

Comment:
DBT_ROOT gets set from dbtenv.cmd, so all uses of SCRIPT_PATH in dbtenv.cmd should just be DBT_ROOT instead, and this won't need to be switched

Reply:
I think we can just leave this as future work, it's better to have something working but inefficient for anyone who tries to download and setup on windows at the moment